models/safe/zksync/safe_zksync_schema.yml

```diff
@@ -0,0 +1,133 @@
+version: 2
+
+models:
+  - name: safe_zksync_safes
+    meta:
+      blockchain: zksync
+      project: safe
+      contributors: ['danielpartida' , 'kryptaki']
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'zksync']
+    description: "Safe addresses"
+    columns:
```

Comment:
```suggestion
    description: "Safe addresses"
    tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - block_date
            - address
    columns:
```

Reply:
Since I cannot directly commit to this PR, I may need your help to update the schema.yml.

Reply:
Sounds good, I'll push the changes this morning